packages/browser-lite/package.json

```diff
@@ -14,14 +15,17 @@
         "docsPath": "../../docs/browser-bindings/browser-lite"
     },
     "dependencies": {
+        "@0x/contract-addresses": "^4.9.0",
```

Comment:
I wonder where this dependency came from? Mesh manages its own contract addresses.

Reply:
Ah, I added this in my PR because I was using contract addresses before the last review. I need to remove this